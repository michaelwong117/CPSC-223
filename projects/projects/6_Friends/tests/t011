#!/bin/bash
# Query all edges

trap "/usr/bin/killall -q -u $USER ./Friends 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./Friends ]; then
  echo './Friends is missing or not executable'
  echo './Friends is missing or not executable' 1>&2
  exit 1
fi


/c/cs474/bin/run -stderr=/dev/null ./Friends /c/cs223/hw6/Tests/graph_alpha_mini_12.in both < /c/cs223/hw6/Tests/t011.in
