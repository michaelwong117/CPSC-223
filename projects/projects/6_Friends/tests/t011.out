A, B have minimum path length 1
A, B have mutual friends C F
B, C have minimum path length 1
B, C have mutual friends A D
A, C have minimum path length 1
A, C have mutual friends B
A, F have minimum path length 1
A, F have mutual friends B
F, B have minimum path length 1
F, B have mutual friends A E
F, E have minimum path length 1
F, E have mutual friends B
E, D have minimum path length 1
E, D have mutual friends B G
B, D have minimum path length 1
B, D have mutual friends C E
C, D have minimum path length 1
C, D have mutual friends B
D, G have minimum path length 1
D, G have mutual friends E
G, E have minimum path length 1
G, E have mutual friends D
B, E have minimum path length 1
B, E have mutual friends D F
